language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -382,7 +382,7 @@ class DottyLanguageServer extends LanguageServer
     else {
       val refs = Interactive.namedTrees(uriTrees, Include.references | Include.overriding, sym)
       (for {
-        ref <- refs
+        ref <- refs if !ref.tree.symbol.isConstructor
```

Comment:
primary constructors are synthetic, but secondary constructors like `def this() = this(1)` are not